### ISSUE TITLE
Crash on CC:T 1.109.3

### ISSUE THREAD

**Issue by SquidDev** [JheffersonMarques/TurtleCosmetics#1]
Your turtle renderer mixin depends on a now-removed method, leading to crashes when running on CC:T 1.109.3:

```
---- Minecraft Crash Report ----
// My bad.

Time: 2024-01-06 21:59:57
Description: Rendering Block Entity

java.lang.NoSuchMethodError: 'net.minecraft.resources.ResourceLocation dan200.computercraft.client.render.TurtleBlockEntityRenderer.getTurtleModel(dan200.computercraft.shared.computer.core.ComputerFamily, boolean)'
	at dan200.computercraft.client.render.TurtleBlockEntityRenderer.render(TurtleBlockEntityRenderer.java:590) ~[cc-tweaked-1.20.1-forge-1.109.3.jar%23615!/:1.109.3] {re:mixin,re:classloading,pl:mixin:APP:TurtleCosmetics.mixins.json:TurtleRenderingMixin,pl:mixin:APP:turtlematic.mixins.json:TurtleBlockEntityRendererMixin,pl:mixin:A}
	at dan200.computercraft.client.render.TurtleBlockEntityRenderer.m_6922_(TurtleBlockEntityRenderer.java:29) ~[cc-tweaked-1.20.1-forge-1.109.3.jar%23615!/:1.109.3] {re:mixin,re:classloading,pl:mixin:APP:TurtleCosmetics.mixins.json:TurtleRenderingMixin,pl:mixin:APP:turtlematic.mixins.json:TurtleBlockEntityRendererMixin,pl:mixin:A}
	at net.minecraft.client.renderer.blockentity.BlockEntityRenderDispatcher.m_112284_(BlockEntityRenderDispatcher.java:90) ~[client-1.20.1-20230612.114412-srg.jar%23958!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:supplementaries.mixins.json:BlockEntityRendererDispatcherMixin,pl:mixin:APP:mixins.oculus.json:entity_render_context.MixinBlockEntityRenderDispatcher,pl:mixin:APP:flywheel.mixins.json:BlockEntityRenderDispatcherAccessor,pl:mixin:APP:embeddiumplus.mixin.json:entitydistance.MaxDistanceTileEntity,pl:mixin:APP:entityculling.mixins.json:BlockEntityRenderDispatcherMixin,pl:mixin:A}
	at net.minecraft.client.renderer.blockentity.BlockEntityRenderDispatcher.m_112297_(BlockEntityRenderDispatcher.java:79) ~[client-1.20.1-20230612.114412-srg.jar%23958!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:supplementaries.mixins.json:BlockEntityRendererDispatcherMixin,pl:mixin:APP:mixins.oculus.json:entity_render_context.MixinBlockEntityRenderDispatcher,pl:mixin:APP:flywheel.mixins.json:BlockEntityRenderDispatcherAccessor,pl:mixin:APP:embeddiumplus.mixin.json:entitydistance.MaxDistanceTileEntity,pl:mixin:APP:entityculling.mixins.json:BlockEntityRenderDispatcherMixin,pl:mixin:A}
	at net.minecraft.client.renderer.blockentity.BlockEntityRenderDispatcher.m_112278_(BlockEntityRenderDispatcher.java:104) ~[client-1.20.1-20230612.114412-srg.jar%23958!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:supplementaries.mixins.json:BlockEntityRendererDispatcherMixin,pl:mixin:APP:mixins.oculus.json:entity_render_context.MixinBlockEntityRenderDispatcher,pl:mixin:APP:flywheel.mixins.json:BlockEntityRenderDispatcherAccessor,pl:mixin:APP:embeddiumplus.mixin.json:entitydistance.MaxDistanceTileEntity,pl:mixin:APP:entityculling.mixins.json:BlockEntityRenderDispatcherMixin,pl:mixin:A}
	at net.minecraft.client.renderer.blockentity.BlockEntityRenderDispatcher.m_112267_(BlockEntityRenderDispatcher.java:79) ~[client-1.20.1-20230612.114412-srg.jar%23958!/:?] {re:mixin,pl:accesstransformer:B,re:classloading,pl:accesstransformer:B,pl:mixin:APP:supplementaries.mixins.json:BlockEntityRendererDispatcherMixin,pl:mixin:APP:mixins.oculus.json:entity_render_context.MixinBlockEntityRenderDispatcher,pl:mixin:APP:flywheel.mixins.json:BlockEntityRenderDispatcherAccessor,pl:mixin:APP:embeddiumplus.mixin.json:entitydistance.MaxDistanceTileEntity,pl:mixin:APP:entityculling.mixins.json:BlockEntityRenderDispatcherMixin,pl:mixin:A}
```

I'm not quite sure why you're using `@Overwrite` here. It would be much better to be using a more precise mixin using `@Inject`.

Originally reported at https://github.com/cc-tweaked/CC-Tweaked/issues/1626#issuecomment-1879902520

**Comment by OHakimen** [comment]
Fixed, and also rewrote the mixin function to be less jank

**Comment by JovannMC** [comment]
Experiencing this same issue because the bug fix update (1.2) wasn't uploaded for Forge 1.20.1, please publish if you can!
